frontend/ios/ScreenshotTests/ScreenshotTests.swift

```diff
@@ -38,49 +38,50 @@ class ScreenshotTests: XCTestCase {
         }
     }
     
+    @MainActor
     func testOpenMap() throws {
         let app = XCUIApplication()
         setupSnapshot(app)
         app.launchArguments += ["UI-Testing"]
         app.launch()
-        
+
         let element = app.staticTexts["Suche\nTab 2 von 4"]
         self.waitForElementToAppear(element: element)
         sleep(60)
-        
         snapshot("01Map")
     }
-    
+
+    @MainActor
     func testOpenSearch() throws {
         let app = XCUIApplication()
         setupSnapshot(app)
         app.launchArguments += ["UI-Testing"]
         app.launch()
-        
+
         let tabItem = app.staticTexts["Suche\nTab 2 von 4"]
         self.waitForElementToAppear(element: tabItem)
         sleep(5)
-        
-        tabItem.tap()
 
+        tabItem.tap()
         snapshot("01Search")
     }
-    
+
+    @MainActor
     func testOpenDetail() throws {
         let app = XCUIApplication()
         setupSnapshot(app)
         app.launchArguments += ["UI-Testing"]
         app.launch()
-        
+
         let tabItem = app.staticTexts["Suche\nTab 2 von 4"]
         self.waitForElementToAppear(element: tabItem)
         sleep(5)
-        
+
         tabItem.tap()
-        let search = app.textFields["Tippen, um zu suchen..."]
+        let search = app.textFields.firstMatch
```

Comment:
i know this is dirty but we use custom unicode characters (short dots) in our translation file and even the fastlane snapshots or the xcode test suite fails if we use the one or the other.
I don't expect we get a second search field on this tab